redbot/cogs/economy/economy.py

```diff
@@ -976,6 +976,7 @@ async def paydayamount(self, ctx: commands.Context, creds: int):
     @economyset.command()
     async def rolepaydayamount(self, ctx: commands.Context, role: discord.Role, creds: int):
         """Set the amount earned each payday for a role.
+        Set to `0` to remove the payday amount you set for that role.
```

Comment:
This needs to also be updated in `docs/cog_guides/economy.rst` file.

Reply:
Resolved in latest PR